services/message.js

```diff
@@ -1,11 +1,22 @@
 const Message = require('~/models/message')
+const Chat = require('~/models/chat')
 
 const messageService = {
   sendMessage: async (author, authorRole, data) => {
-    const { text, isRead, isNotified, chat } = data
+    const { text, chat } = data
 
-    return await Message.create({ author, authorRole, text, isRead, isNotified, chat })
+    const message = await Message.create({
+      author,
+      authorRole,
+      text,
+      chat
+    })
+
+    await Chat.updateOne({ _id: chat }, { latestMessage: message._id })
```

Comment:
i think it would be better to move this updating to post hook of message model as we do it with another models to follow rule that we don't use models in service that are not related to it